src/OpenReader.m

```diff
@@ -480,15 +478,15 @@ -(ASIHTTPRequest*)refreshFeed:(Folder*)thisFolder withLog:(ActivityItem *)aItem
     }
 		
 	NSURL *refreshFeedUrl = [NSURL URLWithString:[NSString stringWithFormat:@"%@stream/contents/feed/%@?client=%@&comments=false&likes=false%@&ck=%@&output=json",APIBaseURL,
-                                                  percentEscape(feedIdentifier),ClientName,itemsLimitation,TIMESTAMP]];
+                                                  percentEscape(feedIdentifier),ClientName,itemsLimitation,[OpenReader currentTimestamp]]];
```

Comment:
Unfortunately you are not meant to use dot notation on class methods. It does work, but xcode will not auto complete and it is frowned upon.